src/JBrowse/Browser.js

```diff
@@ -729,6 +729,9 @@ regularizeReferenceName: function( refname ) {
     if( this.config.exactReferenceSequenceNames )
         return refname;
 
+    // special case of double regularizing behaving badly
+    if(refname === 'chrm') return refname
```

Comment:
Do you think there are other special cases we might catch here? Maybe Plant chloroplasts?

Reply:
It could be worth doing a little survey to see what instances of double regularizing are going wrong. I am not sure if it would be easy to avoid double regularizing

Reply:
It could actually maybe be done to avoid double regularizing but it seems like regularizing something twice still should not cause issues like this